### ISSUE TITLE
ERROR: In the path "obfdist", there are two many files start with "pytransform" or "_pytransform", there shuold be only one

### ISSUE THREAD

**Issue by cpylant** [dashingsoft/pyarmor#503]
It's more readable in the issue Preview mode for the hints

**Hints**
Every pyarmor command prints detail logs in the console, not only check the last error message, but also check each log to understand what pyarmor is doing. And `pyarmor -d subcommand ...` even prints more debug logs, it's very useful to find the problem.

When running the obfuscated scripts, if there is traceback, check the source script according to script name and line number in the traceback, make sure it doesn't use some features changed by pyarmor 

Before report a bug, please read these common questions and solutions first
https://pyarmor.readthedocs.io/en/latest/questions.html

And remove all the above hints.

**Describe the bug**

**To Reproduce**
Which pyarmor version is used, trial or purchased, and steps to reproduce the behavior:
1. Obfuscate the scripts by which Python version and in which platform
2. What command options used
3. How to distribute the obfuscated scripts to target machine if it's different from build machine
4. Run the obfuscated scripts in which platform by which Python version
5. When see error, provide the full traceback and console output by text not snapshot, quote them like this

```
pyarmor obfuscate foo.py
... 
```

**Additional context**

**Comment by jondy** [comment]
Try to remove the whole output path, then pack the scripts again.

**Comment by cpylant** [comment]
I’m not sure which path you mean. ‘dist’, ‘build’, ‘obfdist’?

**Comment by cpylant** [comment]
Here are the commands leading to an obfuscated single-file executable:

```
pyinstaller --icon=v_icon.ico --onefile --noconsole gui.py
pyarmor obfuscate -O obfdist --package-runtime 0 gui.py
python -m pyarmor.helper.repack -p obfdist dist/gui.exe
```
and here is where repack quits with an error:

```
1391 INFO: Patch PYZ done
1399 INFO: Appending runtime files to archive
1399 INFO: Add "obfdist\pytransform.py"
1399 INFO: Add "obfdist\_pytransform.dll"
1399 ERROR: In the path "obfdist", there are two many files start with "pytransform" or "_pytransform", there shuold be only one
```

**Comment by jondy** [comment]
Apply this patch 26d2a982ecf072700a39425b7998d9eec3f2866e to fix it as a quick workaround.

**Comment by cpylant** [comment]
Thanks, this fixed the immediate problem. I find I have to use the "noarchive" debug option with pyinstaller. Is this still secure?

**Comment by jondy** [comment]
`noarchive` doesn't reduce the security more, the structure of pyinstaller archived file is very simple. But I think `repack` should work with pyinstaller one-file mode, if possible, you can check the source of `repack.py` to find the problem, this script is not complex.

**Comment by cpylant** [comment]
Thanks very much for your help.

**Comment by jondy** [comment]
Fix it in v6.7.1
